docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -13,22 +13,20 @@ Users who need a TechPass account but do not have a WOG account or a non-SE GSIB
 
 You need the following to get invited to TechPass and complete the onboarding:
 
-- Your organisational email address provided by your organisation.
-- A standard mailbox for your organisational email address. TechPass does not support email accounts which do not have an inbox, such as LiteMail accounts. If you use such an email account, upgrade it to a standard mailbox before requesting for TechPass.
-- Before you onboard, ensure you have received the TechPass onboarding email and is still valid.
+- Organisational email address with a standard mailbox (not LiteMail).
```

Comment:
what is a standard mailbox?

Reply:
any email which is not a Lite email lah. Ok jokes apart, users need to know this and to focus on the actual prereq i have removed the other parts which were too wordy for a prereq list.

Reply:
sorry i disagree. we can't just remove prerequisites for the sake of making the paragraph shorter. all the prerequisites are important and valid. if you need explanation i can explain further but this, will just make it ambiguous, which is not what we want.

Reply:
@kellyngohhi noted, will revert it.  happy to add any additional info if it makes it clearer to the user. Kindly let me know if we need to define what is a standard mailbox.